### PR TITLE
Update index.ts

### PR DESCRIPTION
**Pull request by mourahi**
no need to import "Host, forwardRef, Provider"

### CODE REVIEW

File: modules/playground/src/person_management/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {bootstrap} from 'angular2/bootstrap';
-import {Component, Directive, View, Host, forwardRef, Provider, Injectable} from 'angular2/core';
+import {bootstrap} from 'angular2/angular2';
```
Comment (IgorMinar) [inline modules/playground/src/person_management/index.ts]: angular2/angular2 will be deprecated soon. we should import this from `angular2/platform/browser`

**Comment by IgorMinar** [comment]
the rest looks good

**Comment by mourahi** [comment]
I 'm not good in english, sorry and thank you for "angular".

the example "person_management", no need to import those class "Host, forwardRef, Provider". that's all.

**Comment by IgorMinar** [comment]
@jelbourn can you please help to fix the commit message before merge? thanks!

**Comment by jelbourn** [comment]
@mourahi looks like this has a failure on Travis

**Comment by IgorMinar** [comment]
it's failing because of the incorrect bootstrap issue I pointed out above.

**Comment by vsavkin** [comment]
@mourahi  could you fix travis?

**Comment by vsavkin** [comment]
Closing this because of inactivity.

**Pull request closed without merging**